### PR TITLE
Add documentation for survey item C14a

### PR DESCRIPTION
**Pull request by nmdefries**
Item C14a replaces item C14 starting in Wave 8.

### CODE REVIEW

**Comment by WichadaLMK** [comment]
C14a, is a revision of item C14, changed from the past 5 days to the past 7 days to be consistent with other items on the COVID Symptom Survey.

File: docs/api/covidcast-signals/fb-survey.md [hunk 1]
```diff
@@ -288,6 +288,7 @@ data in the estimation procedures described above.
 | Signal | Description | Survey Item | Introduced |
 | --- | --- | --- | --- |
 | `smoothed_wearing_mask` | Estimated percentage of people who wore a mask for most or all of the time while in public in the past 5 days; those not in public in the past 5 days are not counted. <br/> **Earliest date available:** 2020-09-08 | C14 | Wave 4, Sept 8, 2020 |
```
Comment (capnrefsmmat) [inline docs/api/covidcast-signals/fb-survey.md:290]: Do we want to add a note to this one indicating it was discontinued in Wave 8?

File: docs/symptom-survey/coding.md [hunk 2]
```diff
@@ -467,6 +467,8 @@ new items were meant to capture reasons for vaccine hesitancy among respondents.
   were revised and expanded for clarity. Please review the changes carefully
   when you use responses from multiple waves of this survey, since they may
   shift which occupations respondents choose.
+* C14a is a revision of item C14, changed from "the past 5 days" to "the past
+  7 days" to be consistent with other items on the COVID Symptom Survey.
```
Comment (capnrefsmmat) [inline docs/symptom-survey/coding.md:471]:
```suggestion
* C14a is a revision of item C14, changed from "the past 5 days" to "the past
  7 days" to be consistent with other items on the COVID Symptom Survey.
  C14a replaces C14.
```

**Review by capnrefsmmat — COMMENTED** [review]
couple suggestions

File: docs/api/covidcast-signals/fb-survey.md (hunk 1 above)
Comment (nmdefries) [inline docs/api/covidcast-signals/fb-survey.md:290]:
```suggestion
| `smoothed_wearing_mask` | *Discontinued as of Wave 8, Feb 8, 2021* Estimated percentage of people who wore a mask for most or all of the time while in public in the past 5 days; those not in public in the past 5 days are not counted. <br/> **Earliest date available:** 2020-09-08 | C14 | Wave 4, Sept 8, 2020 |
```

**Review by capnrefsmmat — APPROVED** [review]
Looks good. Let's merge once we can deploy the new signal

**Comment by capnrefsmmat** [comment]
Deployed; merging.

**Pull request merged**